include/ur_client_library/control/trajectory_point_interface.h

```diff
@@ -49,6 +49,25 @@ enum class TrajectoryResult : int32_t
   TRAJECTORY_RESULT_FAILURE = 2    ///< Aborted due to error during execution
 };
 
+/*!
+ * Spline types
+ */
+enum class TrajectorySplineType : int32_t
+{
+  SPLINE_CUBIC = 0,
+  SPLINE_QUINTIC = 1
```

Comment:
As we have the interfaces for writing position-only trajectory points should we also allow quadratic splines?
```suggestion
  SPLINE_QUADRATIC = 0.
  SPLINE_CUBIC = 1,
  SPLINE_QUINTIC = 2
```